packages/x-components/src/App.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="app">
+    <Tagging :consent="false" />
```

Comment:
Just curious, why are we importing the `Tagging` in the `App.vue`?

Reply:
yes, that's exactly right @CachedaCodes. First was added for local testing but now it is needed also for the E2E test to work properly.